plenum/common/stacks.py

```diff
@@ -160,11 +164,14 @@ def transmitToClients(self, msg: Any, remoteNames: List[str]):
 class NodeZStack(Batched, KITZStack):
     def __init__(self, stackParams: dict, msgHandler: Callable,
                  registry: Dict[str, HA], seed=None, sighex: str=None,
-                 config=None):
+                 config=None, metrics=NullMetricsCollector()):
         config = config or getConfig()
-        Batched.__init__(self, config=config)
+        Batched.__init__(self, config=config, metrics=metrics)
         KITZStack.__init__(self, stackParams, msgHandler, registry=registry,
-                           seed=seed, sighex=sighex, config=config)
+                           seed=seed, sighex=sighex, config=config,
+                           metrics=metrics,
+                           mt_incoming_size=MetricType.INCOMING_NODE_MESSAGE_SIZE,
```

Comment:
Why do we need explicitly pass `mt_incoming_size` and `mt_outgoing_size` params here?